Doc/library/time.rst

```diff
@@ -133,6 +133,7 @@ Functions
 
    .. note::
 
+      The date field is two characters long, and is space padded.
```

Comment:
I suggest to remove this sentence but change the example to use a day which shows the space padding:  ``'Sun Jun  5 23:21:05 1993'``

Reply:
@vstinner: I can update the example in the current doc, but I think that it will difficult to notice an extra space in the example unless mentioned specifically or we mention strftime() format.

Reply:
@pganssle I have added the example with both the possible outcomes. Let me know if something needs to be changed.